root/static/scripts/common/i18n/localizeLanguageName.js

```diff
@@ -0,0 +1,24 @@
+/*
+ * @flow strict
+ * Copyright (C) 2021 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+function localizeLanguageName(
+  language: LanguageT | null,
+  isWork?: boolean,
+): string {
+  if (!language) {
+    return l('[removed]');
+  }
+  // For works, "No linguistic content" is meant as "No lyrics"
+  if (isWork /*:: === true */ && language.iso_code_3 === 'zxx') {
```

Comment:
You can default `isWork` to false so that it's not undefined if not specified:

```suggestion
  isWork?: boolean = false,
): string {
  if (!language) {
    return l('[removed]');
  }
  // For works, "No linguistic content" is meant as "No lyrics"
  if (isWork && language.iso_code_3 === 'zxx') {
```

Reply:
True, true, I keep forgetting about this about half the time, sigh! Done :)